components/HotspotsList.js

```diff
@@ -108,8 +113,9 @@ const HotspotsList = ({ hotspots, rewardsLoading, hotspotsLoading }) => {
             loading={hotspotsLoading}
             size="small"
             rowKey="name"
-            pagination={{ pageSize: 10, hideOnSinglePage: true }}
+            pagination={{ pageSize }}
```

Comment:
Is there a reason why `hideOnSinglePage: true` was removed?

```suggestion
            pagination={{ pageSize, hideOnSinglePage: true }}
```

Reply:
Thanks for pointing this out. Actually, with `hideOnSinglePage: true` the dropdown gets disappeared if all the values are displayed. I feel this would be problematic for some users that's why I removed it. 

As you can see in the screen capture on displaying all the data, the dropdown has vanished and the user doesn't have an option to reduce the table rows.

![helium-table](https://user-images.githubusercontent.com/42844688/107775574-7ad80000-6d66-11eb-8f2b-de05be27f7a6.gif)

Reply:
Oh, that makes perfect sense, thanks!

Reply:
good catch! 😄